source/includes/connection-snippets/scram-snippets/nodejs-connection.js

```diff
@@ -0,0 +1,10 @@
+// begin scram connection
```

Comment:
for our recent documentation work we've been using a more complex template using async/await:
```
const { MongoClient } = require('mongodb');

const client = new MongoClient(
  'mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority'
);

async function run() {
  try {
    await client.connect();
    const collection = client.db('test').collection('devices');
    // perform actions on the collection object
  } finally {
    // Ensures that the client will close when you finish/error
    await client.close();
  }
}

run().catch(console.dir);
```

Reply:
Fixed - similar to my comment below: copied basically verbatim and removed the `const collection = client.db('test').collection('devices');` line since it's not needed for this ticket